src/components/tracked-controls.js

```diff
@@ -18,7 +19,10 @@ module.exports.Component = registerComponent('tracked-controls', {
     controller: {default: 0},
     id: {type: 'string', default: ''},
     idPrefix: {type: 'string', default: ''},
-    rotationOffset: {default: 0}
+    rotationOffset: {default: 0},
+    // Arm model parameters, to use when not 6DOF. (pose hasPosition false, no position)
+    headElement: {type: 'selector'},
+    defaultHand: {type: 'string', default: 'right'}
```

Comment:
so internalize defaultHand, ok makes sense (although maybe should be configurable somehow like on tracked-controls system?) as just discussed.  am currently fighting with windows 10 update, if you guys want to do that before I can get to it, go right ahead